SYCL/lit.cfg.py

```diff
@@ -176,37 +176,52 @@
     config.sycl_be = config.sycl_be[3:]
 config.sycl_be = config.sycl_be.lower()
 
+# Replace deprecated backend names
+deprecated_names_mapping = {'cuda'       : 'ext_oneapi_cuda',
+                            'hip'        : 'ext_oneapi_hip',
+                            'level_zero' : 'ext_oneapi_level_zero',
+                            'esimd_cpu'  : 'ext_intel_esimd_emulator'}
```

Comment:
```suggestion
                            'esimd_cpu'  : 'ext_intel_esimd_emulator'}
```
```suggestion
                            'esimd_emulator'  : 'ext_intel_esimd_emulator'}
```

Reply:
here old name of the ESIMD emulator was used https://github.com/intel/llvm/blob/sycl/sycl/include/CL/sycl/backend_types.hpp#L31